test/spec/modules/lotamePanoramaIdSystem_spec.js

```diff
@@ -446,4 +446,117 @@ describe('LotameId', function() {
       'lotamePanoramaId': '1234'
     });
   });
+
+  describe('with an empty cache, ignore profile id for error 111', function () {
+    let request;
+    let callBackSpy = sinon.spy();
+
+    beforeEach(function () {
+      let submoduleCallback = lotamePanoramaIdSubmodule.getId({}).callback;
+      submoduleCallback(callBackSpy);
+
+      request = server.requests[0];
+
+      request.respond(
+        200,
+        responseHeader,
+        JSON.stringify({
+          profile_id: '4ec137245858469eb94a4e248f238694',
```

Comment:
in practice, this response would never come back. If error 111 were returned, it wouldn't have a panorama id. Doesn't hurt to have the test case, but just letting you know it should never happen

Reply:
ok